src/Bicep.Core/Emit/EmitHelpers.cs

```diff
@@ -15,18 +15,11 @@ public static class EmitHelpers
         /// </summary>
         /// <param name="resourceSymbol">The resource symbol</param>
         /// <exception cref="ArgumentException">If the resource symbol is not for a valid resource type.</exception>
-        public static ResourceTypeReference GetTypeReference(ResourceSymbol resourceSymbol)
-        {
+        public static ResourceTypeReference GetTypeReference(ResourceSymbol resourceSymbol) =>
```

Comment:
>ResourceSymbol [](http://example.com/codeflow?start=61&length=14)

Should we make it an extension method? Or a method on `ResourceSymbol`?

Reply:
Moved it to `ResourceSymbol`.